usagov/vote-gov
# Feature/vote 1783 dependabot update

ticket -> [Vote1783](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/266?modal=detail&selectedIssue=VOTE-1783&assignee=611272e0eb7aef00694b8a97)

This PR will update from dependabot updates 
will close: https://github.com/usagov/vote-gov/pull/884, https://github.com/usagov/vote-gov/pull/878 and https://github.com/usagov/vote-gov/pull/876

Note - gulp stylelint and stylelint scss were removed due to not being maintained, the gulp.js file was updated from uswds example to help cover the new listing function 

to test:

- pull down branch 
- run `npm run test` and check that linting is catching files that need to be updated 
- run `npm run start` and ensure app is starting with no issues
